src/dataProcessor/dataProcessorService.js

```diff
@@ -1 +1,46 @@
+let getDebt = (
+  currentLeasePayable,
+  currentOtherFinancialLiabilities,
+  nonCurrentLeasePayable,
+  nonCurrentOtherFinancialLiabitlies
+) =>
+  currentLeasePayable +
+  currentOtherFinancialLiabilities +
+  nonCurrentOtherFinancialLiabitlies +
+  nonCurrentLeasePayable;
 
+let getEBIT = (profit, interestIncome, interestExpense, tax) =>
+  profit + tax + interestExpense - interestIncome;
+
+let getCaptialEmployed = (debt, equity) => debt + equity;
+
+let getROCE = (EBIT, captialEmployed) => EBIT / captialEmployed;
+
+let getPERatio = (stockPrice, EPS) => stockPrice / EPS;
```

Comment:
Why is these _get_ functions defined as an arrow function while `processRawData` is using `function` keyword? What is the decision behind using these 2 different styles for these functions?